automation/taskcluster/androidTest/parse-ui-test.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+
+import sys
+import argparse
+from pathlib import Path
+import json
+import yaml
```

Comment:
Bug: 

```
[task 2020-10-13T10:10:41.597Z] + chmod +x ./automation/taskcluster/androidTest/parse-ui-test.py
[task 2020-10-13T10:10:41.598Z] + ./automation/taskcluster/androidTest/parse-ui-test.py --exit-code 0 --log flank.log --results /builds/worker/artifacts/results --output-md /builds/worker/artifacts/github/customCheckRunText.md --device-type arm
[task 2020-10-13T10:10:41.623Z] Traceback (most recent call last):
[task 2020-10-13T10:10:41.623Z]   File "./automation/taskcluster/androidTest/parse-ui-test.py", line 7, in <module>
[task 2020-10-13T10:10:41.623Z]     import yaml
[task 2020-10-13T10:10:41.623Z] ModuleNotFoundError: No module named 'yaml'
[task 2020-10-13T10:10:41.626Z] + exit 0
[fetches 2020-10-13T10:10:41.626Z] removing /builds/worker/fetches
```

https://firefox-ci-tc.services.mozilla.com/tasks/XOXO6SmpSdiPfmsT22tZSA/runs/0/logs/https%3A%2F%2Ffirefox-ci-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FXOXO6SmpSdiPfmsT22tZSA%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log

You can add `RUN pip install pyyaml` there https://github.com/mozilla-mobile/android-components/blob/97c612f9f41f183b3edc5167d1a6a09cf9ab97ab/taskcluster/docker/ui-tests/Dockerfile#L13